content/docs/people/p32/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: P32
+description: >
+  This page contains information about P32.
+confidentiality: public
+---
+This is the placeholder for the documentation of P32. As the entire concept is still wip, only current google docs will be linked. In the future you can find a proper documentation.
+
+*Information Sources:*
+* [Slack Channel](https://app.slack.com/client/T0251EQJH/C0430C5BD9N?cdn_fallback=1)
+* [Success Indicators](https://docs.google.com/document/d/1O4MzuQk6FejgfY_FjA76Ku7Lfo4ea5qgCcXkcTVlMFE/edit)
```

Comment:
Cut the `/edit` from the `Success Indicators` and the `?cdn_fallback=1` from the Slack link?

Maybe we should always do this kind of cleanup?